### DISCUSSION TITLE
Fraunhofer FDK AAC decoder

### DISCUSSION THREAD

**Discussion by Ant-12** [earlephilhower/ESP8266Audio#537] [Ideas]
Is it possible to add this AAC decoder?
[https://github.com/pschatzmann/arduino-fdk-aac](https://github.com/pschatzmann/arduino-fdk-aac)

**Comment by earlephilhower** [comment]
That's nice, and the author has something similar to this library which includes the same kind of processing.

However, I'm wondering what you are expecting different between the Real Networks AAC decoder already here and this one.  The main memory requirements are probably the same, because it's part of the standard.

**Comment by Ant-12** [comment]
I expect the completely decoding of HE-AACv2 streams, including the "Parametric Stereo" feature.

**Comment by Ant-12** [comment]
**[earlephilhower](https://github.com/earlephilhower),** do I understand correctly that you are satisfied with [the lack of stereo output](https://github.com/earlephilhower/ESP8266Audio/blob/c2adbdcf887fec65d25af0e5b8b47ae424df54e4/src/libhelix-aac/readme.txt#L22) when decoding [HE-AACv2](https://en.wikipedia.org/wiki/High-Efficiency_Advanced_Audio_Coding) streams and are not going to implement ["Parametric Stereo"](https://en.wikipedia.org/wiki/Parametric_Stereo) [support](https://github.com/pschatzmann/arduino-fdk-aac/blob/b10f230e9218663274faff6c95868264f7ac1efa/src/libSBRdec/psbitdec.cpp#L444) ?

**Reply by earlephilhower** [reply]
I wouldn't say that, but I do have other things occupying my time.  If you'd like to do a PR that would be great!  Otherwise, it will have to wait its turn in the queue...
